tests/mentorship_relation/test_api_list_relations.py

```diff
@@ -171,7 +171,7 @@ def test_list_current_mentorship_relation_sent_by_another_user(self):
     def test_list_filter_mentorship_relations_empty(self):
         with self.client:
             response = self.client.get(
-                "/mentorship_relations?relation_state=%s" % "",
+                "/mentorship_relations?relation_state= ",
```

Comment:
```suggestion
                "/mentorship_relations?relation_state=",
```
The tests are failing because of one of my suggestions, I think I fixed it here. Can you apply this suggestion please?

Reply:
@isabelcosta The checks have passed successfully now. Thanks !